examples/tf/optimization_advanced_tf.py

```diff
@@ -36,11 +36,11 @@
 def circuit():
 
     # This time we want to keep the parameter small via regularization and visualize its evolution in tensorboard
-    alpha = make_param(name='alpha', constant=0.1, regularize=True, monitor=True)
+    params = [make_param(name='alpha', constant=0.1, regularize=True, monitor=True)]
     eng, q = sf.Engine(1)
 
     with eng:
-        Dgate(alpha) | q[0]
+        Dgate(params[0]) | q[0]
```

Comment:
Writing the parameter into a list and then unpacking works as well, but I found it instructive to show that with the TensorFlow graph, one can explicitly address parameters by name...On the other hand, it's closer to the numerical learner. @josh, what do you think?